agent/agent.go

```diff
@@ -234,6 +236,11 @@ func (a *orbAgent) RestartBackend(ctx context.Context, name string, reason strin
 	if err := a.policyManager.RemoveBackendPolicies(be, true); err != nil {
 		a.logger.Error("failed to remove policies", zap.String("backend", name), zap.Error(err))
 	}
+	configuration := structs.Map(a.config.OrbAgent.Otel)
+	configuration["agent_tags"] = a.config.OrbAgent.Tags
+	if err := be.Configure(a.logger, a.policyManager.GetRepo(), a.config.OrbAgent.Backends["pktvisor"], configuration); err != nil {
```

Comment:
remove 'pktvisor' reference to avoid breaking backend abstraction, were else document this as tech debt with a plan to fix

Reply:
done